config/runtimes/vllm-multinode-template.yaml

```diff
@@ -177,15 +201,33 @@ objects:
           emptyDir:
             medium: Memory
             sizeLimit: 12Gi
+        - name: ray-ca-cert
+          secret:
+            secretName: ray-ca-cert
+        - name: ray-tls
+          emptyDir: {}
+        # The gencert_ray.sh can be prebaked into the docker container so the configMap is optional
+        - name: gen-tls-script
```

Comment:
what happen if user wants to use their own certs?

Reply:
It is for internal communication so I don't consider the use case.
If there is a request for this, we can work it at that time.